resource/src/utils/model.mustache

```diff
@@ -19,6 +19,26 @@ exports.create = (modelName) => {
   model.standardQuery = exports.queryBuilder(model, modelSchema);
   model.standardFeed = exports.feedBuilder(model);
 
+  if (opts.audit) {
+    const auditModel = thinky.createModel(
+      modelSchema.pluralName + '_audit_log',
```

Comment:
this will mix camelcase with snakecase, propbably should be something like `+ 'AuditLog'`